test/foundry/new/helpers/FuzzMutations.sol

```diff
@@ -5,17 +5,126 @@ import { Test } from "forge-std/Test.sol";
 import { FuzzExecutor } from "./FuzzExecutor.sol";
 import { FuzzTestContext } from "./FuzzTestContextLib.sol";
 
-contract FuzzMutations is Test, FuzzExecutor {
+import { AdvancedOrder } from "seaport-sol/SeaportStructs.sol";
+
+import { OrderType } from "seaport-sol/SeaportEnums.sol";
+
+import { AdvancedOrderLib } from "seaport-sol/SeaportSol.sol";
+
+import { LibPRNG } from "solady/src/utils/LibPRNG.sol";
+
+contract MutationFilters {
+    using AdvancedOrderLib for AdvancedOrder;
+
+    // Determine if an order is unavailable, has been validated, has an offerer
+    // with code, has an offerer equal to the caller, or is a contract order.
+    function ineligibleForInvalidSignature(
+        AdvancedOrder memory order,
+        uint256 orderIndex,
+        FuzzTestContext memory context
+    ) internal view returns (bool) {
+        if (!context.expectedAvailableOrders[orderIndex]) {
+            return false;
+        }
+
+        if (order.parameters.orderType == OrderType.CONTRACT) {
+            return false;
+        }
+
+        if (order.parameters.offerer == context.caller) {
+            return false;
+        }
+
+        if (order.parameters.offerer.code.length != 0) {
+            return false;
+        }
+
+        (bool isValidated, , , ) = context.seaport.getOrderStatus(
+            context.orderHashes[orderIndex]
+        );
+
+        if (isValidated) {
+            return false;
+        }
+
+        return true;
+    }
+}
```

Comment:
Aren't these booleans flipped?  Or does something down stream make it make sense?

Seems like the name `ineligibleForInvalidSignature` should return `true` in cases where we're NOT going to be able to trigger a failure by passing in an invalid sig.